aquia-inc/template-vite-react
# build(dev-deps): upgrade storybook minor versions

## Summary

Upgraded the Storybook-related devDependencies to their latest versions to ensure compatibility, improved performance, and access to new features.

### Added

- None

### Changed

- Upgraded `@storybook/addon-essentials` from 7.6.16 to 7.6.20
- Upgraded `@storybook/addon-interactions` from 7.6.16 to 7.6.20
- Upgraded `@storybook/addon-links` from 7.6.16 to 7.6.20
- Upgraded `@storybook/blocks` from 7.6.16 to 7.6.20
- Upgraded `@storybook/client-api` from 7.6.16 to 7.6.20
- Upgraded `@storybook/react` from 7.6.16 to 7.6.20
- Upgraded `@storybook/react-vite` from 7.6.16 to 7.6.20

### Deprecated

- None

### Removed

- None

### Fixed

- None

## How to test

1. Checkout this branch: `git checkout origin/build/dependencies/storybook`
2. Run `yarn` and ensure all dependencies are correctly installed without duplication and that there are no issues related to missing or incompatible dependencies.
5. Run `yarn storybook` and verify that the Storybook application runs as expected.
6. Run `yarn build-storybook` and verify that the Storybook build succeeds.

## Related Issues/PRs

- Resolves #39
